iphone/Classes/ContactsModule.m

```diff
@@ -716,12 +719,16 @@ -(TiContactsPerson*)createPerson:(id)arg
 		}
 		NSError *error = nil;
 		CNMutableContact *newContact = [[CNMutableContact alloc] init];
-		TiContactsPerson* newPerson = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext] contactId:newContact module:self] autorelease];
+		TiContactsPerson* newPerson = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext]
```

Comment:
This change removes the required arguments ` contactId:newContact module:self`. Did you test the changes?